L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorSuperTriggerCellImpl.cc

```diff
@@ -130,7 +130,7 @@ void HGCalConcentratorSuperTriggerCellImpl::assignSuperTriggerCellEnergyAndPosit
     double denominator = fixedDataSizePerHGCROC_
                              ? double(kTriggerCellsForDivision_)
                              : double(superTCmapping_.getConstituentTriggerCells(stc.getSTCId()).size()) /
-                                   double(coarseTCmapping_.getConstituentTriggerCells(stc.getSTCId()).size());
+      ( coarsenTriggerCells_ ? (double(coarseTCmapping_.getConstituentTriggerCells(stc.getSTCId()).size())) : 1. );
```

Comment:
It's becoming a bit hard to read this line, with two ternary operators on the same line. Could it be split in 2 statements?